crates/flux-infer/src/fixpoint_encoding.rs

```diff
@@ -159,6 +160,13 @@ pub mod fixpoint {
                 DataSort::Adt(adt_id) => {
                     write!(f, "Adt{}", adt_id.as_u32())
                 }
+                // There's no way to declare opaque sorts in the fixpoint horn syntax so we encode user
+                // declared opaque sorts, type parameter sorts, and (unormalizable) type alias sorts as
+                // integers. Well-formedness should ensure values of these sorts are used "opaquely",
+                // i.e., the only values of these sorts are variables.
```

Comment:
This comment is wrong; it should only refer to opaque sorts. Also, sort parameters and (unnormalizable) type alias sorts are still encoded as `int` in `sort_to_fixpoint`, so part of this comment should go there.